MANIFEST.in

```diff
@@ -1,10 +0,0 @@
-include CONTRIBUTING.md
```

Comment:
manifest.in isn't really needed much these days (except in complicated cases),  it's mostly a relic of distutils